code/game/machinery/autolathe_datums.dm

```diff
@@ -456,3 +456,55 @@
 	name = "miniature cell"
 	path = /obj/item/weapon/cell/device/emergency_light/empty
 	category = "Engineering"
+	
+/datum/autolathe/recipe/cable_coil
+	name = "cable coil"
+	path = /obj/item/stack/cable_coil
+	category = "Devices and Components"
+	is_stack = 1
+
+/datum/autolathe/recipe/labeler
+	name = "hand labeler"
+	path = /obj/item/weapon/hand_labeler
+	category = "General"
+
+/datum/autolathe/recipe/destTagger
+	name = "destination tagger"
+	path = /obj/item/device/destTagger
+	category = "General"
+
+/datum/autolathe/recipe/implanter
+	name = "implanter"
+	path = /obj/item/weapon/implanter
+	hidden = 1
+	category = "Medical"
+	
+/datum/autolathe/recipe/debugger
+	name = "debugger"
+	path = /obj/item/device/debugger
+	category = "Engineering"
+	
+/datum/autolathe/recipe/micromanip
+	name = "micro-manipulator"
+	path = /obj/item/weapon/stock_parts/manipulator
+	category = "Devices and Components"
+	
+/datum/autolathe/recipe/matterbin
```

Comment:
This is included in the protolathe, no need for the autolathe to have it.

Reply:
Yeah it was just the very basic stock parts that were added, it was an idea I just stole from bay because it's very handy for engineers when science isn't kicking about (scanning mods for camera upgrades, etc). 

All the better parts like (advanced, pic, super, etc) are still protolathe only. Got a lot of conflicting opinions on whether basic stock parts should be printable, also the parts in tech storage isn't enough to even make an autolathe.